custom_md_loader/classes/MetadataLoaderClient.cls

```diff
@@ -0,0 +1,73 @@
+/* 
+ * Copyright (c) 2016, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license. 
+ * For full license text, see LICENSE.txt file in the repo root  or https://opensource.org/licenses/BSD-3-Clause
+ */
+ 
+public class MetadataLoaderClient {
```

Comment:
Can you please add class-level comments to indicate that this class is a usage example?